vignettes/programming_strategy.Rmd

```diff
@@ -5,8 +5,11 @@ output:
     toc: true
 vignette: >
   %\VignetteIndexEntry{Programming Strategy}
-  %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  markdown: 
+    wrap: 72
```

Comment:
I don't mind adding this as it'll clean up the look for developers moving forward, but makes this current PR a little more difficult to review

Reply:
I agree.  Sophie can you revert this change and just show the updates for the missing values.  

Perhaps a separate pr for this wrap could be used if folks want this for easier viewing??

Reply:
Of course, no problem - didn't realise this had been altered! Apologies